doc/sphinxman/CMakeLists.txt

```diff
@@ -57,10 +57,10 @@ if(PERL_FOUND AND SPHINX_FOUND AND SPHINX_STUFF_FOUND)
     manage_addon.rst numpy.rst build_planning.rst build_faq.rst
```

Comment:
Just a general question: what do you think about having one source file per line (here and in other `CMakeLists.txt`)? It should make it easier to parse diffs.

Reply:
`tests/CMakeLists.txt` is the choicer target